lte/gateway/python/scripts/dhcp_helper_cli.py

```diff
@@ -75,16 +76,20 @@ class DhcpHelperCli:
     _SNIFFER_STARTUP_WAIT = 0.5
     _TIMEOUT = 10
     _state = DHCPState.DISCOVER
-    _msg_xid = 0
+    _msg_xid = random.randint(0, 2**32 - 1)
```

Comment:
Why is random needed here? Why 2^32-1?

Reply:
On every call of the `dhcp_helper_cli`, a new instance gets created. If we leave the `_msg_xid` on any given number here, it will always be the same, and we cannot discern between different packets with it (i.e. the `_msg_xid` will be rather pointless in that case). We take a random integer to solve this issue. We chose 2^32-1 because the xid in the scapy `BOOTP` packet is a 32 bit field.